sample/src/main/java/vehicles/AbstractVehicle.java

```diff
@@ -0,0 +1,40 @@
+package vehicles;
+
+import people.Person;
+
+import java.time.LocalDate;
+
+/**
+ * An abstract class for implementing different kinds of {@link Vehicle}.
```

Comment:
Particularly I think the javadoc @link tag is nice and easy to write and it is extremely useful inside IDEs to navigate through the documentation, since asciidoc link are not parsed by IDEs.
But on the other hand, such javadoc tags will be exported as is to the adoc file and will not be parsed when generating the final documentation in PDF or any other format.

I think that avoiding the second issue may be better. However, I'm wondering if we could address the first issue and if you have some directions.